<?xml version="1.0" encoding="UTF-8"?>
<launch>
<arg name="bino_stereo_sdk" default="$(env BINO_STEREO_SDK)"/>
	<node name="bino_stereo" pkg="bino_stereo_ros" type="bino_stereo_ros" output="screen" >
		<param name="framerate_"        value="60"/>  <!--FPS-->
		<param name="video_device" 	    value="/dev/video1" />  <!--摄像头设备号-->
		<param name="camera_left_frame_id"   value="bino_left_frame" /> 
		<param name="camera_right_frame_id"   value="bino_right_frame" /> 
		<param name="intrinsuc_file"       type="string"      value="$(arg bino_stereo_sdk)/params/intrinsics.yml"/> 
		<param name="extrinsic_file"       type="string"      value="$(arg bino_stereo_sdk)/params/extrinsics.yml"/> 
		<!--remap from="/bino_camera/left/image_rect"  to="/bino_camera/left/image_rect_bino"/>
		<remap from="/bino_camera/right/image_rect"  to="/bino_camera/right/image_rect_bino"/-->
		<remap from="/bino_camera/ImuData"  to="/imu/data_raw"/>
	</node>
	<arg name="pi/2" value="1.5707963267948966" />
	<!--arg name="optical_rotate" value="0.12 0.06 0.55 -$(arg pi/2) 0 -$(arg pi/2)" /-->
	<arg name="optical_rotate" value="0.00 0.00 0.00 -$(arg pi/2) 0 -$(arg pi/2)" />
	<!--arg name="optical_rotate" value="0 0 0 -$(arg pi/2) 0 -$(arg pi/2)" /-->
	<node pkg="tf" type="static_transform_publisher" name="camera_link"
		args="$(arg optical_rotate) camera_base bino_left_frame 100" /> 
	<node pkg="tf" type="static_transform_publisher" name="camera_rotate"
		args="0 0 0 0 0.26 0 rotate_link camera_base 100" /> 
	<node pkg="tf" type="static_transform_publisher" name="rotate_base_link"
		args="0.12 0.07 0.66 0 0 0 base_link rotate_link 100" /> 
</launch>
